ui/src/app/domain/employee/model/Emplyee.ts

```diff
@@ -0,0 +1,6 @@
+export interface emplyee{
+    id:number;
+    name:string;
+  
```

Comment:
from swagger you can get the object of employee property 
where property of company?

![image](https://user-images.githubusercontent.com/96727804/200284438-1287933d-0aa9-4ca3-ab62-301e98be6114.png)